ports/nrf/boards/pca10059/mpconfigboard.h

```diff
@@ -71,3 +71,5 @@
 #endif
 
 #define HELP_TEXT_BOARD_LED         "1,2,3,4"
+
+#define MICROPY_PY_SYSTICK          (1)
```

Comment:
It would be nice if this is put on top with the other `MICROPY_PY_*` definitions.

Reply:
done